### PR TITLE
Added missing "Code is Poetry" block in some packages

### PR DESCRIPTION
**Pull request by mukeshpanchal27**
## Description
Some of the package documents of gutenberg handbook are missing "Code is Poetry" block at the end of the page.

### CODE REVIEW

File: packages/wordcount/README.md [hunk 1]
```diff
@@ -27,3 +27,5 @@ count accepts three parameters:
 ```JS
 import { count } from '@wordpress/wordcount';
 const numberOfWords = count( 'Words to count', 'words', {} )
+
+<br/><br/><p align="center"><img src="https://s.w.org/style/images/codeispoetry.png?1" alt="Code is Poetry." /></p>
```
Comment (aduth) [inline packages/wordcount/README.md]: This does not display correctly, as the prior code block is not terminated.

**Review by aduth — CHANGES_REQUESTED** [review]

File: packages/wordcount/README.md (hunk 1 above)
Comment (aduth) [reply packages/wordcount/README.md]: There should only be three back ticks to end the block. It does seem to display correctly in the preview though, which is unexpected to me.

Comment (mukeshpanchal27) [reply packages/wordcount/README.md]: @aduth thanks for your guidance

**Review by aduth — APPROVED** [review]
Thanks 👍

**Pull request merged**